metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/database/SraObjectsImporterThroughDBTest.java

```diff
@@ -135,11 +142,17 @@ public void importAnalysisObjectWithEmptyReferenceSequenceAccession() throws Exc
 
     @Test
     @Category(OracleDbCategory.class)
-    public void importSampleObject() throws Exception {
-        Sample sample = sraObjectImporter.importSample("ERS000002");
-        assertNotNull(sample);
-        assertEquals("ERS000002", sample.getAccessionVersionId().getAccession());
-        assertEquals("Solexa sequencing of Saccharomyces cerevisiae strain SK1 random 200 bp library", sample.getName());
+    public void importSamplesObject() throws Exception {
+        SraXmlParser<AnalysisType> analysisTypeSraXmlParser = new SraAnalysisXmlParser();
+        String xmlString = new String(Files.readAllBytes(
+                Paths.get(getClass().getClassLoader().getResource(ANALYSIS_DOCUMENT_BIG_XML).toURI())));
+        AnalysisType analysisType = analysisTypeSraXmlParser.parseXml(xmlString, "ERZ015710");
+        List<Sample> sample = sraObjectImporter.importSamples(analysisType);
```

Comment:
can you do the import again in this same function and check the sample count is same

Reply:
Re-verifying both count and a value

Reply:
can you please verify the sample repository count